src/components/course/enrollment/tests/EnrollAction.test.jsx

```diff
@@ -176,7 +176,15 @@ describe('scenarios user not yet enrolled, but eligible to enroll', () => {
     expect(screen.getByText(enrollLabelText).closest('button')).toBeInTheDocument();
     expect(screen.getByText(enrollLabelText).closest('a')).not.toBeInTheDocument();
     const regex = new RegExp().compile(ENROLL_MODAL_TEXT_NO_OFFERS);
+    screen.debug();
     expect(screen.getByText(regex)).toBeInTheDocument();
+
+    const PAYMENT_TEXT = 'Continue to payment';
+    // also check url is rendered in the modal correctly
```

Comment:
Personally, I would make this a separate test. That way when the test fails, the dev can know exactly what's going wrong, instead of having to wade through various expects in one test.